app/controllers/accountants_controller.rb

```diff
@@ -13,9 +13,9 @@ def search
     # We're doing it janky because we implemented search to return an array,
     # not an activerecord object. some room for improvement.
     @results = if params[:search].present?
-                 Patient.search(params[:search])
-                        .select(&:pledge_sent?)
-                        .sort_by(&:pledge_sent_at).reverse
+                 Patient.where(pledge_sent: true)
+                        .order_by(pledge_sent_at: :desc)
+                        .search(params[:search])
                else
```

Comment:
This is basically undoing a change some clever dude made two years ago. He wrote the comment above on lines 13-14 suggesting we couldn't return active record objects. 

https://github.com/DCAFEngineering/dcaf_case_management/commit/0714914b6dc94beb2d7ba4c32f02346d3134d171#diff-4f7fab0f140af92109627f65edf210c3R15-R21

That said, it does seem to be functional now for me in dev, with 1 results, 0 results, and multiple results. So if we're happy with returning this object, can we delete the comment?

Reply:
definitely!

So my explanation now that I understand the finer points of how activerecord works: When we hit `search` we return a real array because of some concatenation we do, but `search` starts with activerecord, so we can feed activerecord into it, and it'll still do a proper query.

I will delete the comment!